release_notes/40-release-notes.adoc

```diff
@@ -0,0 +1,444 @@
+:_content-type: ASSEMBLY
+[id="release-notes-40"]
+= Red Hat Advanced Cluster Security for Kubernetes 4.0
+include::modules/common-attributes.adoc[]
+:context: release-notes-40
+
+toc::[]
+
+{product-title} ({product-title-short}) is an enterprise-ready, Kubernetes-native container security solution that protects your vital applications across build, deploy, and runtime stages of the application lifecycle. It deploys in your infrastructure and integrates with your DevOps tools and workflows to deliver better security and compliance and to enable DevOps and InfoSec teams to operationalize security.
+
+.Release dates
+[options="header"]
+|====
+|{product-title-short} version |Released on
+|`4.0` |1 May 2023
+|====
+
+[id="about-this-release-40"]
+== About this release
+
+{product-title-short} 4.0 includes the following new features, improvements, and updates:
+
+* <<major-release-version-change>>
+* <<external-postgresql-database-rn40>>
+* <<acs-cloud-release-rn40>>
+* <<telemetry-rn40>>
+* <<rhcos-node-scan-rn40>>
+* <<process-listening-endpoints-rn40>>
+* <<network-graph-rn40>>
+* <<fips-compliance-rn40>>
+* <<token-expiry-alert-rn40>>
+* <<sensor-resync-rn40>>
+
+[id="new-features-40"]
+== New features
+
+[id="major-release-version-change"]
+=== Major release version change to 4.0
+With this release, {product-title-short} includes a major architecture change, moving the Central database to PostgreSQL. This change provides important performance and scale 	benefits. Central is now faster and can scale to support larger data stores.
+
+By default, {product-title-short} installation includes the PostgreSQL database. Future plans include the ability to use your own PostgreSQL-compliant software. {product-title-short} 4.0 includes this feature as a Technology Preview with certain limitations. For more information, see "Installing Central with an external PostgreSQL database (Technology Preview)".
+
+{product-title-short} includes features that depend on the new architecture. Specifically, policy criteria and vulnerability reporting with collections were introduced in 3.74 as a Technology Preview, and are now generally available.
+
+//update link to Customer Portal link after CP version is published (lags behind by ~1 day)
+For instructions for a new installation of {product-title-short} 4.0, see xref:../installing/acs-installation-platforms.adoc[Supported platforms and installation methods].
+
+[id="upgrade-to-40"]
+=== Upgrading {product-title-short} to version 4.0
```

Comment:
This part turns to be very operator-centric and does not give details about upgrades for Helm and `roxctl`-based installs.

I would suggest you put in the end the same links that the part for ACSCS has.

```
- xref:../upgrading/upgrade-operator.adoc[Upgrading by using the Operator]
- xref:../upgrading/upgrade-helm.adoc[Upgrading using Helm charts]
- xref:../upgrading/upgrade-roxctl.adoc[Manually upgrading using the roxctl CLI]
```

It would be nice to also say that Helm and `roxctl`-based upgrades are same as before except that Central database backup is recommended prior to upgrading Central.

Reply:
I moved the docs listed section around a bit and tried to add this language. It's awkward because we aren't allowed to say we "recommend" something, or that you "should" do something, per IBM & RH Style guidelines.